frontend/src/pages/Causes.jsx

```diff
@@ -1,19 +1,37 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./styles/Causes.scss";
 import FilterTab from "../components/FilterTab";
 import ProfileCard from "../components/ProfileCard";
+import axios from "axios";
 
 function Causes() {
+  const [associations, setAssociations] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:4000/association`)
```

Comment:
Recommendation: you shouldvariabilize this (conf.js and dotenv), if you leave it this way it will break when we deploy this to production